docs/UserGuide.md

```diff
@@ -1,19 +1,51 @@
-# User Guide
+# WerkIt! User Guide
 
 ## Introduction
 
 {Give a product intro}
 
+---
+
 ## Quick Start
 
 {Give steps to get started quickly}
 
 1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+2. Down the latest version of WerkIt! from [here (WIP)](http://link.to/duke).
+3. (Recommended) Create a new directory and move the WerkIt! JAR file to the directory.
+4. (Recommended) Set your current working directory to the new directory.
+5. Run the WerkIt! application with the command: `java -jar WerkIt.jar`.
+
+---
 
 ## Features 
 
-{Give detailed description of each feature}
+> **Heads up!** Your inputs cannot contain the pipe character `|`!
+
+### Create a Workout: `workout /new`
+Creates a new workout.
+
+Format: `workout /new <exercise name> /reps <number of repetitions>`
+
+| Parameters | Description                                                                                                                                                      |
+| --- |------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `<exercise name>` | The name of the exercise. <ul><li>Exercise name must be from the pre-defined list of exercises</li><li>Type `exercise /list` for the list of exercises</li></ul> |
+| `<number of repetitions>` | The number of repetitions that should be carried out for `<exercise name>`.                                                                                      |
+
+**Example**
+```
+> workout /new russian twist /reps 50
+```
+**Expected Outcome**
+```
+----------------------------------------------------------------------
+Alright, the following workout has been created:
+
+	russian twist (50 reps)
+
+----------------------------------------------------------------------
+```
+A new workout of carrying out Russian twists 50 times will be created and added to the application's list of workouts.
 
 ### Adding a todo: `todo`
```

Comment:
Remember to remove template

Reply:
Thanks for reminding me. It has since been removed in Commit babb92d.